packages/lodestar/src/network/gossip/gossipsub.ts

```diff
@@ -1,4 +1,7 @@
-import Gossipsub, {IGossipMessage, Options, Registrar} from "libp2p-gossipsub";
+import PeerId from "peer-id";
+import Gossipsub from "libp2p-gossipsub";
+import {Registrar} from "libp2p-gossipsub/src/peer";
+import {Message} from "libp2p-gossipsub/src/message";
```

Comment:
is it ok to import like this or gossipsub has to publish `Message` directly?

Reply:
Should be ok to import like that. We can make sure to export `Message` to root for the 0.29 migration